pom.xml

```diff
@@ -361,6 +361,8 @@
                     <excludes>
                         <!-- ignore until #2601 is resolved -->
                         <exclude>jmri.jmrit.decoderdefn.DuplicateTest</exclude>
+                        <!-- ignore until #2671 is resolved -->
+                        <exclude>jmri.jmrix.roco.z21.simulator.Z21SimulatorAdapterTest</exclude>
```

Comment:
Please do not exclude this test here.

If it is being excluded, we can't see that it fails.  It only fails on Travis Linux in headless mode, and I can't replicate that locally.  It also doesn't happen frequently enough for me to catch it before someone restarts a build, so I won't know if the problem is fixed until it runs for an extended period of time.

Also, that is not the only test class that might be causing the failure.  (This isn't a 
problem with the test, it's a threading issue with the underlying code that occurs
when the connection is terminated).

Paul